mmocr/models/textrecog/encoders/positional_encoder.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import math
+
+import torch
+from torch import nn
+
+from mmocr.models.builder import ENCODERS
+from .base_encoder import BaseEncoder
+
+
+@ENCODERS.register_module()
+class PositionalEncoder(BaseEncoder):
+    """Implementation of position encoder module in `MASTER.
+
+    <https://arxiv.org/abs/1910.02562>`_.
+
+    Args:
+        d_model (int): Dim :math:`D_i` of channels from backbone.
+        dropout (float): Dropout probability in encoder.
+        max_len (int): max length of feature sequence,
+                    should be greater or equal to flatten feature map's length.
+    """
+
+    def __init__(self, d_model, dropout=0., max_len=5000):
```

Comment:
```suggestion
    def __init__(self, d_model, dropout=0., max_seq_len=5000):
```
The naming can be consistent across modules